RangeShiftR/src/Rinterface.cpp

```diff
@@ -1614,7 +1627,7 @@ int ReadStageStructureR(Rcpp::S4 ParMaster)
 				ss = (float)trmatrix(i - 1, i); // survival prob
 			else
 				ss = (float)trmatrix(i, i);
-			if((i + 2) != matrixsize)
+			if((i + 2) != matrixsize && (i + 1) != matrixsize) // was if((i + 2) != matrixsize); but what happens in the last column (matrixsize -1)? (i+1) would be out of bounds
```

Comment:
forgot to fix this bug already in main -> did the bugfix now in develop and main